local-gems/spectrum-json/lib/spectrum/entities/alma_item.rb

```diff
@@ -1,3 +1,4 @@
+require 'byebug'
```

Comment:
I'm going to remove this one `require 'byebug'` line before merging.  But everything looks good to me.